fluidreview/views.py

```diff
@@ -13,6 +14,10 @@ class WebhookView(APIView):
     permission_classes = (WebhookPermission,)
     authentication_classes = ()
 
+    def handle_exception(self, exc):
+        """Raise any exception with request info instead of returning response with error status/message"""
+        raise FluidReviewException("REST Error (%s). BODY: %s, META: %s", exc, self.request.body, self.request.META)
```

Comment:
The `%s` values aren't getting replaced with anything here

Reply:
✅